src/app/services/raiden.service.ts

```diff
@@ -415,19 +417,19 @@ export class RaidenService {
     }
 
     public swapTokens(swap: SwapToken): Observable<boolean> {
-        const data = {
+        const body = losslessStringify({
```

Comment:
Do you need to do the conversion by hand at every call? Couldn't you use an interceptor to do the conversion?

Reply:
Yes, that is easily possible. Thanks!